examples/hello_world.rs

```diff
@@ -10,30 +10,39 @@ fn main() {
 }
 
 fn setup(mut commands: Commands) {
-    commands.spawn(Camera3dBundle::default());
+    commands.spawn(Camera3d::default());
     // `spawn_task` is a helper to spawn an async task onto the Bevy executor with a
     // `TaskContext` instance to get mutable world access
     commands.spawn_task(|cx| async move {
         // `TaskContext` provides APIs built on top of `with_world` to simplify common tasks
 
         // For example, `TaskContext::spawn` is shorthand for
         // `cx.with_world(|world| world.spawn(...).id())`
-        let e = cx
-            .spawn(
-                TextBundle::from_section(
-                    "Waiting for keyboard event",
-                    TextStyle {
-                        font_size: 36.0,
-                        ..default()
-                    },
-                )
-                .with_style(Style {
-                    align_self: AlignSelf::Center,
-                    justify_self: JustifySelf::Center,
-                    ..default()
-                }),
-            )
+        let container = cx
+            .spawn(Node {
+                justify_self: JustifySelf::Center,
+                align_self: AlignSelf::Center,
+                ..default()
+            })
             .await;
+
+        // We can't use `cx.spawn` here because we need to use `set_parent`
+        // Improved constructors in bevy 0.16 sould make this more ergonomic
```

Comment:
The explanation here are helpful! I'm going to spin this out into an issue for Bevy 0.16 so it doesn't drop off the radar.